src/components/Inputs/TextInput/TextInput.js

```diff
@@ -141,6 +148,11 @@ function PrivateTextInput({
 PrivateTextInput.PUBLIC_PROPS = {
   'data-tid': PropTypes.string.isRequired,
   disabled: PropTypes.bool,
+  // This is a newer way to trigger the mobile numeric keypad (old way was
+  // `type='tel'`.) For now, we probably either want to leave this unset, in
+  // which case it is equivalent to 'text', or set 'numeric' for numeric inputs.
+  inputMode: PropTypes.oneOf(Object.values(INPUT_MODES)),
```

Comment:
I doubt this broke anything, but did you happen to spot check zip, bd, and textinput in the styleguidist based DS?